docs/openapi.yaml

```diff
@@ -552,55 +662,88 @@ paths:
-    categoryParam:
-      name: category
+    abstractParam:
+      name: abstract
+      in: query
+      description: "returns document with..."
+      required: false
+      schema:
+        type: string
+    abstractNotParam:
+      name: abstract_not
+      in: query
+      description: "returns document with..."
+      required: false
+      schema:
+        type: string
+    abstractShouldParam:
+      name: abstract_should
       in: query
       description: "Returns documents with this category code, only one category code is allowed."
       example: "JournalArticle"
       required: false
       schema:
         $ref: "#/components/schemas/CategoryEnum"
-    categoryNotParam:
-      name: category_not
+    contextTypeNotParam:
+      name: context_type_not
       in: query
-      description: "Returns documents without these category codes. Support list of Enums, seperated by comma (+)."
-      example: "JournalArticle"
+      description: "returns document without these contextTypes, separated by comma."
       required: false
       schema:
-        $ref: "#/components/schemas/CategoryEnum"
-    categoryShouldParam:
-      name: category_should
+        type: string
+    contextTypeParam:
+      name: context_type
       in: query
-      description: "Returns documents with these category codes. Support list of Enums, seperated by comma (+)."
-      example: "JournalArticle"
+      description: "returns document with these contextTypes, separated by comma."
       required: false
       schema:
-        $ref: "#/components/schemas/CategoryEnum"
-    contributorIdParam:
-      name: contributor_id
+        type: string
+    contextTypeShouldParam:
+      name: context_type_should
+      in: query
+      description: "?"
+      required: false
+      schema:
+        type: string
+    contributorNameNotParam:
+      name: contributor_name_not
+      in: query
+      description: "returns document without Author's name"
+      required: false
+      schema:
+        type: string
+    contributorNameParam:
+      name: contributor_name
       in: query
-      description: "person id (URI)."
+      description: "returns document with Author's name "
+      required: false
+      schema:
+        type: string
+    contributorNameShouldParam:
+      name: contributor_name_should
+      in: query
+      description: "returns document with similar Author names"
       required: false
       schema:
         type: string
     contributorParam:
       name: contributor
       in: query
-      description: "returns document with Author's name or person id(URI)."
+      description: "returns document with Author's id(URI)."
       required: false
       schema:
         type: string
     contributorNotParam:
       name: contributor_not
       in: query
-      description: "returns document without Author's names or person ids(URI), seperated by comma (+)."
+      description: "returns document without Author's ids(URI), separated by comma."
```

Comment:
Does this one accept comma separated list? We don't specify that for other `not`'s